### DISCUSSION TITLE
Blank bar and spinning icon

### DISCUSSION THREAD

**Discussion by snowdog850** [valinet/ExplorerPatcher#3121] [General]
I have ExplorerPatcher installed on two separate computers. On one of them today, the whole bar is black, except for the windows icon on the left and the settings icon on the right. However, when I mouse over the icons (or anywhere on the bar), the mouse just turns to the windows spinning icon. I cannot start any applications. I cannot even use win+r or win+e to open anything. I can do ctrl-alt-del to pull up Task Manager. But nothing else.

Any idea what's going on? More importantly, any idea how I can uninstall or update ExplorerPatcher when I can't seem to get to anything at all?

**Comment by bbmaster123** [comment]
in task manager, first kill explorer.exe if its still running, then go to run new task, type in control and hit enter. go to programs and features, find explorerpatcher, uninstall

If for whatever reason you cant load up control panel, go to task manager > run new task > browse, go to C:\Windows and rename or delete dxgi.dll, then load up explorer.exe

After you get back to a working state, update windows and download the latest explorerpatcher which works on up to stable build 22631.3447.

**Reply by snowdog850** [reply]
Ugh. It appears unrelated to ExplorerPatcher. I was able to uninstall following your directions. But now the bar appears along the bottom of my screen (I usually have it set to the top). But still black with no icons. 

I rolled back the latest Windows Update, I did a system restore to 3 days ago. Nothing helps. When I boot to safe mode I get the Windows taskbar. 

Anyway, I know this is not Explorer Patcher now, but if anyone else has any ideas, let me know. I’d rather not reinstall Windows.

**Reply by bbmaster123** [reply]
glad you could get back to a semi working state, but that's still frustrating!
have you tried the 3 dism commands followed by sfc /scannow (maybe a chkdsk C: /x /f /r too) and a reboot? do you have any errors in your event viewer? and lastly, have you tried resetting your taskbar in regedit? To do that, export and then delete Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\StuckRects3 in registry editor. Once StuckRects3 is gone, sign out and in or reboot, and the taskbar SHOULD be reset to the default working windows 11 taskbar

If for any reason running those commands doesn't help and you have no errors in event viewer and resetting the taskbar doesn't work, you can try an in place reinstall from the settings app. If that also fails, maybe you could run memtest and check your bios but other than that I would think that's all you can do.

Hopefully you don't have to follow all of those steps, good luck! Let me know if you need help with any of those steps!

**Reply by snowdog850** [reply]
Since I was able to get the taskbar back, I restarted without any services. It worked. I am now in the process of rebooting while enabling chunks of services each time to narrow it down to which service is causing the problem.

**Comment by pyrates999** [comment]
If you need to repair your windows install, run the following in an elevated command prompt or elevated powershell:

```
chkdsk C:
```

If it tells you there are errors, then run this:

```
chkdsk C: /F
```

Reboot to run it on boot.  Keep running that command until it reports no errors.  Then run this to repair windows:

```
sfc /scannow
dism /online /cleanup-image /scanhealth
dism /online /cleanup-image /restorehealth
dism /online /cleanup-image /checkhealth
dism /online /cleanup-image /analyzecomponentstore
dism /online /cleanup-image /startcomponentcleanup
```

Then reboot your computer.

**Comment by snowdog850** [comment]
OK, I've identified the service causing the problem. It's "WLAN Autoconfig". No clue why that service would be affecting my taskbar, but hey, it's Windows.

I'll try the suggestions here to see if there's corruption somewhere that can be fixed.

Thanks for all of the help!
